test/stubs/clinician_input_stub.c

```diff
@@ -9,24 +9,44 @@
 #include "platform/common.h"
 
 static uint32_t* RR_to_return;
-static status_t* status_to_return;
-static int current_call = 0;
-
-
+static status_t* status_RR_to_return;
+static int RR_current_call = 0;
 
 void set_respiratory_rate(uint32_t* RR, status_t* status, size_t size){
   free(RR_to_return);
-  free(status_to_return);
+  free(status_RR_to_return);
```

Comment:
Not a fan of the dynamic  memory management in the test code... you could revise this.

Reply:
part of another merge request for RR